tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -54,17 +54,17 @@ def test_add_node(self):
                 )
 
         elif config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM:
-            pytest.skip(
-                "Skipping add node in Vmware platform due to "
-                "https://bugzilla.redhat.com/show_bug.cgi?id=1844521"
+            logger.info(
+                f"The worker nodes number before expansion {len(node.get_worker_nodes())}"
             )
-            # Issue to remove skip code https://github.com/red-hat-storage/ocs-ci/issues/2403
-            # logger.info(f'The worker nodes number before expansion {len(node.get_worker_nodes())}')
-            # if config.ENV_DATA.get('rhel_user'):
-            #     pytest.skip("Skipping add RHEL node, code unavailable")
-            # node_type = constants.RHCOS
-            # assert add_new_node_and_label_upi(node_type, new_nodes), "Add node failed"
-            # logger.info(f'The worker nodes number after expansion {len(node.get_worker_nodes())}')
+            if config.ENV_DATA.get("rhel_user"):
+                pytest.skip("Skipping add RHEL node, code unavailable")
```

Comment:
I think same `add_new_node_and_label_upi()` can be used here, only `node_type` will be different for RHEL. 
Isn't it?

Reply:
Yes, I think you right.